Salesforce.gitignore

```diff
@@ -0,0 +1,53 @@
+# GitIgnore for Salesforce Projects
+# Project Settings and MetaData
+.project
+.settings/
+.metadata
+build.properties
+config
```

Comment:
Hey @jamesward - love to get your thoughts on this, I have https://github.com/mailtoharshit/gitignore project seeking community input, which I merged it here